plugins/GSdx/GSSettingsDlg.cpp

```diff
@@ -155,6 +155,7 @@ void GSSettingsDlg::OnInit()
 	ComboBoxInit(IDC_CRC_LEVEL, theApp.m_gs_crc_level, theApp.GetConfig("crc_hack_level", 3));
 
 	CheckDlgButton(m_hWnd, IDC_PALTEX, theApp.GetConfig("paltex", 0));
+	CheckDlgButton(m_hWnd, IDC_LARGE_FB, theApp.GetConfig("large_framebuffer", 0));
```

Comment:
Personally I prefer having it disabled by default, but I see that gregory has it enabled on linux GUI. so went with that.